examples/remix/README.md

```diff
@@ -0,0 +1,186 @@
+# Use with Remix
+
+This example demonstrates how to use `jest-preview` with [`Remix`](https://remix.run/), bootstrapped using [`create-remix`](https://remix.run/docs/en/v1#getting-started) (without using any built-in templates/stacks).
```

Comment:
I think we can centralize the guide to www.jest-preview.com. We can move this tutorial to `website/docs/examples` and refer the link here.

Reply:
I just feel like keep it in the `examples` folder makes more sense 😂
Let it be there for now.